images/prow-tests/README.md

```diff
@@ -1,6 +1,7 @@
 # Prow Test Job Image
 
-This directory contains the custom Docker image used by our Prow test jobs.
+This directory contains the custom Docker image used by our Prow test jobs, with
```

Comment:
Revert this change, add the note about using Go 1.12 to prow-tests-go112/README.md